lib/kube/proxy/exec_test.go

```diff
@@ -137,6 +137,15 @@ func TestExecKubeService(t *testing.T) {
 				config: configWithSingleKubeUser,
 			},
 		},
+		{
+			name: "Websocket protocol v5",
+			args: args{
+				executorBuilder: func(c *rest.Config, s string, u *url.URL) (remotecommand.Executor, error) {
+					return newWebSocketClient(c, s, u, withSupportedProtocols([]string{v5BinaryWebsocketProtocol}))
```

Comment:
```suggestion
					return remotecommand.NewWebSocketExecutor(c, s, u.String())
```

use kubectl own implementation. The `newWebSocketClient` as something that I custom built because at the time there was no websocket client for kubernetes in go.